### ISSUE TITLE
Docs Request: CodeSandbox and/or Storybook Demo

### ISSUE THREAD

**Issue by GeorgeWL** [dev-saeed/neumorphic-ui#2]
Just think it would be a nice way to see the components in action, as if they're in the real world.

**Comment by lyqht** [comment]
I agree with this suggestion! Or minimally there should be screenshots on how the components look like before the user can decide to try out this library 😄

**Comment by dev-saeed** [comment]
Good suggestion. For now, I would add screenshots so that users may have a visual idea of the components. After that, I would be adding sandboxes for live demo.
